WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -0,0 +1,49 @@
+// Workaround for https://github.com/cubing/icons/issues/8
+
+.cubing-icon.icon-mmagic::before {
+  content: 'mmagic';
+}
+.cubing-icon.icon-magic::before {
+  content: 'magic';
+}
+.cubing-icon.icon-333mbo::before {
+  content: '333mbo';
+}
+
+.event-checkbox {
+  label {
+    margin-bottom: 0px;
+  }
+  input[type=checkbox] {
+    display: none;
+    & + span.cubing-icon {
+      color: #ccc;
```

Comment:
I've added `cursor: pointer` in 5b99de5e573a2a9f492e4d204fd0c2f63fc86e5e. I didn't do it at first because there isn't a pure css solution for this, AFAIK.